pt-PT.json

```diff
@@ -3,300 +3,521 @@
+	"STORAGE_FULL_TEXT": "Não será possível guardar as definições ou informações de conta. Considera fechar o Stremio e voltar a abrir, caso este erro persista, poderás ter de reinstalar o Stremio.",
+	"SEARCH": "Pesquisar",
 	"SEARCH_NO_RESULTS": "Não foram encontrados resultados",
 	"SEARCH_EXPLANATION_CONTENT": "Pesquisa por filmes, séries, canais YouTube e de TV",
-	"SEARCH_EXPLANATION_PEOPLE": "Pesquisa por atores, diretores e escritores",
+	"SEARCH_EXPLANATION_PEOPLE": "Pesquisa por atores, realizadores e argumentistas",
 	"SEARCH_PASTE_LINKS": "Colar links HTTP e Magnet",
-	"ADD_TO_LIB": "Ad. à biblioteca",
-	"REMOVE_FROM_LIB": "Rem. da biblioteca",
-	"ADDED_TO_LIB": "Adicionado à Biblioteca",
-	"REMOVED_FROM_LIB": "Removido da Biblioteca",
+	"SEARCH_OR_PASTE_LINK": "Pesquisa ou cola o link",
+	"ADD_TO_LIB": "Adicionar à biblioteca",
+	"REMOVE_FROM_LIB": "Remover da biblioteca",
+	"ADDED_TO_LIB": "Adicionado à biblioteca",
+	"REMOVED_FROM_LIB": "Removido da biblioteca",
 	"TRAILER": "Trailer",
-	"WATCH_TRAILER": "Ver Trailer",
-	"WATCH_NOW": "Ver Agora",
+	"WATCH_TRAILER": "Ver trailer",
+	"WATCH_NOW": "Ver agora",
 	"SHOW": "Mostrar",
-	"WATCH_RANDOM": "Ver aleatoriamente",
-	"IMDB_RATING": "Classif. IMDb",
+	"SHOW_MOVIE": "Mostrar filme",
+	"WATCH_RANDOM": "Ver aleatório",
+	"IMDB_RATING": "Classificação IMDb",
 	"YEAR": "Ano",
-	"DIRECTOR": "Diretor",
-	"WRITER": "Escritor",
-	"LEAD_ACTORS": "Atores Principais",
+	"DIRECTOR": "Realizador",
+	"WRITER": "Argumentista",
+	"LEAD_ACTORS": "Elenco Principal",
 	"CAST": "Elenco",
-	"CREW": "Crew",
+	"CREW": "Equipa Técnica",
 	"SHOW_MORE_CAST": "Mostrar mais »",
 	"AIRED": "Transmissão original",
 	"SEASONS": "Temporadas",
 	"GENRE": "Género",
-	"CATALOG": "Catalog",
+	"CATALOG": "Catálogo",
 	"SUMMARY": "Sinopse",
+	"SHOW_MORE": "Mostrar mais",
 	"SIMILAR": "Semelhante",
 	"UPCOMING": "Próximos",
 	"ENDED": "Finalizados",
 	"DURATION": "Duração",
 	"FEATURED": "Destaque",
-	"METADATA": "Metadata",
-	"CATEGORY": "category",
+	"METADATA": "Metadados",
+	"CATEGORY": "categoria",
+	"LINKS_GENRES": "Géneros",
+	"LINKS_CAST": "Elenco",
+	"LINKS_DIRECTORS": "Realizadores",
 	"SEARCH_VIDEOS": "pesquisar vídeos",
-	"SEARCH_PLACEHOLDER": "search or paste magnet link",
-	"ADDON": "Extensão",
-	"ADDONS": "Extensões",
+	"SEARCH_PLACEHOLDER": "pesquisar ou colar link",
+	"ADDON": "Addon",
+	"ADDONS": "Addons",
 	"OFFICIAL": "oficial",
-	"ADDONS_OFFICIAL": "Extensões Oficiais do Stremio",
-	"ADDONS_POPULAR": "Popular na sua área",
-	"ADDON_FREE": "Gratuito",
+	"ADDONS_OFFICIAL": "Addons oficiais do Stremio",
+	"ADDONS_POPULAR": "Popular na tua área",
+	"ADDON_FREE": "gratuito",
 	"ADDON_PEERED": "partilhado",
-	"ADDON_SUBSCRIPTION": "assinatura",
-	"ADDON_STREAM_GEORESTRICTED": "não disponível na sua área",
-	"ADDON_STREAM_OFFLINECACHE": "Cache offline",
+	"ADDON_SUBSCRIPTION": "subscrição",
+	"ADDON_STREAM_GEORESTRICTED": "indisponível no teu país",
+	"ADDON_STREAM_OFFLINECACHE": "Cache Offline",
```

Comment:
The text on this line is not translated, but the first letter of "O" is made uppercase compared to the english text, it should be corrected by changing back to lowercase "o" letter for "offline"